tests/lib/rules/capitalized-comments.js

```diff
@@ -40,6 +40,7 @@ ruleTester.run("capitalized-comments", rule, {
         "//\xDCber",
         "//\u03A0",
         "/* Uppercase\nsecond line need not be uppercase */",
+        "// ",
```

Comment:
Sounds good to me. Added. Thanks. (I have the branch set up so that ESLint Members can push directly to it, so if other desirable whitespace tests come to mind, feel free to add them yourself if that's easier/faster.)